brokerapi/brokers/account_managers/sql_account_manager.go

```diff
@@ -69,9 +70,9 @@ func (sam *SqlAccountManager) CreateCredentials(instanceID string, bindingID str
 
 	var creds SqlAccountInfo
 	// create ssl certs
-        certname := bindingID + "cert"
+	certname := bindingID + "cert"
 	if len(bindingID) >= 10 {
```

Comment:
It feels like this hard-coded length and subsequent slice range could benefit for a constant.

Reply:
I'll add a ticket for these because they're all over the place, service account lengths, project name lengths, etc. At least the service account is labeled with a BUG tag, but I don't know if it was ever updated or planned to be.